R/convert_to_numeric.R

```diff
@@ -4,12 +4,13 @@
 #' converted, the target columns are the ones returned by the \code{scan_data()}
 #' function. Furthermore, it identifies columns where the proportion of numeric
 #' values is at least twice the percentage of character values and performs the
-#' conversion in them.
+#' conversion in them. The function internally makes call of the main function
+#' from the {numberize}.
```

Comment:
Can use `\pkg{numberize}` for formatting.

Reply:
I accounted for this